config/config-sample.json

```diff
@@ -9,5 +9,5 @@
   "FS_HOSTNAME" : "",
   "UI_INTERFACE" : "0.0.0.0",
   "UI_PORT" : 3000,
-  "UI_JWT_SECRET": ""
+  "UI_JWT_SECRET": "your_secret"
```

Comment:
just for debug purpose no need any manually configuration. In production mode we can change it to some real value

Reply:
Use --ui-jwt-secret command line argument for development. Also, the config sample will be removed because the config file is now deprecated in favor of using environment variables.